### PR TITLE
Feature disk space

### PR DESCRIPTION
**Pull request by ximex**
Adds a progress-bar of the used disk space

### CODE REVIEW

File: adm_program/system/function.php [hunk 1]
```diff
@@ -697,6 +697,48 @@ function admFuncGetDirectoryEntries($directory, $searchType = 'file')
 }
 
 /**
+ * Gets the total, free and used disk space.
+ * @param string $path Path of the filesystem
+ * @return array<string,int>
```
Comment (Fasse) [inline adm_program/system/function.php]: Please add a description here how the array will be filled with each value. Maybe additional an example.

File: adm_program/system/function.php [hunk 2]
```diff
@@ -697,6 +697,48 @@ function admFuncGetDirectoryEntries($directory, $searchType = 'file')
...
+    $used = $total - $free;
+
+    return array('total' => $total, 'free' => $free, 'used' => $used);
+}
+
+/**
+ * Returns a human readable size with unit
+ * @param int  $size The size in bytes
+ * @param bool $si   Use SI or binary unit. Set true for SI units
+ * @return string
```
Comment (Fasse) [inline adm_program/system/function.php]: Please add a description here how the array will be filled with each value. Maybe additional an example.

File: adm_program/system/function.php (hunk 1 above)
Comment (Fasse) [reply adm_program/system/function.php]: It's a short function and you may see it in the code but you should also have in mind that we can create a documentation without the code and than you don't know how the array looks like.

File: adm_program/system/function.php (hunk 2 above)
Comment (ximex) [reply adm_program/system/function.php]: which array? I think you mean the format of the return string

Comment (Fasse) [reply adm_program/system/function.php]: ok, copy paste thing. I mean the return string.

**Comment by ximex** [comment]
@Fasse there is only a problem if we have a so long text with the min-width. Look here: https://getbootstrap.com/docs/3.3/components/#progress-label

**Comment by Fasse** [comment]
Ok, I understand. But maybe we should rename it to "Speicherplatz" anyway?

**Pull request merged**